### ISSUE TITLE
Mondo mappings should be sorted to reduce diff

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo#4751]
All sssom tables when run through split_sssom_by_source should be sorted entirely (like by all columns) and the columns should also be sorted in the sssom preferred order.

Right now, some rows in pandas seem to randomly fluctuate.

**Comment by matentzn** [comment]
Even better, implement https://github.com/mapping-commons/sssom-py/issues/51

can call ssom-py function in split_sssom_by_source

**Comment by hrshdhgd** [comment]
Done: https://github.com/mapping-commons/sssom-py/issues/51

**Comment by matentzn** [comment]
in mondo makefile?

**Comment by hrshdhgd** [comment]
Yes: https://github.com/monarch-initiative/mondo/blob/master/src/ontology/mondo.Makefile#L471

**Comment by matentzn** [comment]
This sorts mondo.sssom.TSV, what about the files that are split out in the python script?

**Comment by hrshdhgd** [comment]
https://github.com/monarch-initiative/mondo/pull/5491
